node/src/client.rs

```diff
@@ -1,3 +1,4 @@
+use cumulus_primitives_proof_size_hostfunction::storage_proof_size::HostFunctions as ProofSize;
```

Comment:
Why do we have Cumulus dependencies?

Reply:
> Why do we have Cumulus dependencies?

Because the benchmark were failing due to a missing host function

Reply:
Hmm, it weird but the frontier example also contains it: https://github.com/polkadot-evm/frontier/blob/c6c059c4d4aa1fbb586506a471c31bd9351dc13f/template/node/src/service.rs#L48

Reply:
Yes, it seems that they used it for non-benchmark compilation too, on our case, it was only failing when compiling for benchmarks